entgql/internal/todo/ent/schema/todo.go

```diff
@@ -69,7 +69,9 @@ func (Todo) Edges() []ent.Edge {
 			Annotations(entgql.Bind()).
 			Unique(),
 		edge.From("category", Category.Type).
+			Annotations(entgql.BindDisabled()).
 			Ref("todos").
+			Annotations(entgql.BindDisabled()).
```

Comment:
This is a demonstration for using `BindDisabled()`, it keeps the generated code is the same as before.